lib/ui/painting.dart

```diff
@@ -1574,6 +1574,10 @@ class Paint {
 
 /// The format in which image bytes should be returned when using
 /// [Image.toByteData].
+///
+/// NOTE: This ImageByteFormat enum needs to converge and will not add more
```

Comment:
Nit: `s/NOTE://`

(and below)

Reply:
I don't think it needs to converge (we're not going to reduce the values in the enum) -- just say not to add new values to the enum.

Reply:
+1 - we should just make this a `//` comment, and ideally link to the github issue history for context.

Reply:
Done! Thanks